### PR TITLE
Makes radbow mute and decreases price

### PR DESCRIPTION
**Pull request by ShadowDeath6**
title

people apparently didn't like it being outclassed by the revolver in every aspect, why blame em

:cl: ShadowDeath6
rscadd: The radiation crossbow now mutes its victims
tweak: The radiation crossbow now costs 11 TC, down from 12
/:cl:

### CODE REVIEW

**Comment by Time-Green** [comment]
So
A weapon that insta-stuns, mutes has infinite ammo and deals rad damage.

You know it can pretty much perma-stun someone now, without them screaming and just keep clicking till they die.

I don't know if that's balanced or not, but it seems ridiculous.

**Comment by ShadowDeath6** [comment]
revolver fires faster and kills in less shots
you be the judge

**Comment by Time-Green** [comment]
Ergh, true. 
Let's just make it spit mutagen instead, that'll be balanced :D

**Comment by Super3222** [comment]
Eh, I suggest not making it automatically mute on the first shot. Maybe perhaps
`if(C.confused)
    C.silent += 3`

So it'd require you to at least sink in that first shot, and make it less than an infinite-charging long-distant parapen.

**Comment by ShadowDeath6** [comment]
what would that code actually accomplish though? it confuses on the first shot

**Comment by Super3222** [comment]
Put it in place before line 170, so the code doesn't confuse itself. This implies that we already shot the target once, and are then attempting to shoot them a second time.

**Comment by ShadowDeath6** [comment]
ah I see, thanks

**Comment by ParagonPrime** [comment]
I almost never see radbow used, so gud change.

**Comment by FluffySurvivor** [comment]
radbow is extremely powerful and fun when used properly

I once destroyed the lights in the hallways and maint, bought thermals and started shooting people from the darkness. I eventually sparked a huge manhunt in maint and managed to kill every hostile, one by one, some sick plays there. 

ahhh good ol antag freedom times where you wouldn't get banned for killing antaghunters

nostalgia aside, this is a good change. The price reduction was needed, but the mute might prove to be a little bit too strong, we'll see.

**Comment by ShadowDeath6** [comment]
The point is to make this a decent alternative to guns because lelhowevermanyclicks2crit will always be better than a shit stun with slow recharge. Hopefully the mute makes it slightly more viable in actually being lethal instead of being shot once and saying "L-E-E EB-BOW MA-I-N-T-T" when you get shot

**Comment by Alek2ander** [comment]
Suggestion: remove stun and mute and make ebow completely silent (no messages to victim)

**Comment by ghost** [comment]
Alex, radbow was changed significantly. The stun is extremely short and it does not weaken-it stuns, so they're still standing up, just can't move.

Also, one shot from the radbow will not only do significant tox damage, but the radiation is enough to trigger horrible mutations-meaning even one shot will assfuck the victim with tourettes if they're unlucky.

I'm fine with a price reduction but not a mute. Making the attack completely silent is also a bad idea-i'd say make the message smaller so it's not immediately noticeable.

**Comment by Alek2ander** [comment]
I think making the attack silent isn't that bad. You can't spam shoot the ebow, you can see the inhand/projectile if you pay attention, and you'll notice your health decreasing/confusion making your movement weird.

**Comment by ParagonPrime** [comment]
Confusion might actually be a bad idea- people normally move in straight, predictable ways and confusion would actually make it harder to land shots.

Then again, I guess that would be H U G B O X

**Comment by Super3222** [comment]
Alright, so I took the radbow out for a test run during a traitor round... from what I can say being in close range  within a closed area will make it very easy to kill a target. Sorta of like a revolver, but, it's much slower to fire. Within a fair amount of shots into my target, they eventually got knocked out or their stamina is drained. Something like that. I'm very sneaky with my shots, and like to gamble with situations here and there. But I'll let out everything from my personal opinion and experience playing the game.

Now at a fairly estimated medium range, it took me a few shots to finally get them to collapse. Maybe it was because they were wearing an engineering hardsuit, maybe not. But, if another traitor hadn't emagged telecoms then I would've been exposed pretty badly. He was screaming, A LOT.

From what I discovered, the radbow within the first 2-3 shots will almost always lock them in a position for death. Whether it be from the radiation making them exhausted (and falling down to collapse) or whether it'd be another last shot to cause the former. Honestly, I'd think it'd be a little more RNG honestly depending on the time frame of your shots (on whether they hit or miss).

I'm not going to fabricate anything here, this weapon is **pretty goddamn lethal**. But let's take a look at how Shadow is buffing it.
He's adding a muting effect, which can only happen if the target is already confused.
Confusion can happen a lot of ways, drunk people, certain chemicals, maybe brain damage.
But where it's most likely going to come from, is the first ebow shot from the gun.

So let's say we're trying to kill some random greytider (definitely not a breach of the rules) just sitting around.

So when that first shot connects, the guy is already confused and he's moving all over the place. If he's lucky, maybe he'll move in a direction to get him away from the shooter. In the time frame of the ebow recharging it self, he has enough time to scream over the comms. Meaning whether they'll have time to scream for help or not will only depend on how they would prioritize their time between the ebow-recharge time (which is decent).

Your question is probably "but what's the point of this weapon even, and why is it this way even?"
The entire point at the beginning was to get rid of the combo cafe cheese.
Making the ebow more lethal by introducing a new weapon that was outside of chemicals, which could toxicate it's target's was fascinating.
At this point, the radbow is basically just an alternative of the revolver. It's a weapon which to use, you have to gamble with it and play your cards right.

We can't sport how well this gun is already on practical RNG, that is unfair. We have to look at what it's suppose to do, and how people are going to use it. And what I see is a gun which gives you promising results in it's own way.

So let's get back to my main point here ... **the radbow will only mute targets which are already confused**. 

Now, while we're looking at the predicted gameplay strats here, let's compare it to the revolver.

One shot from a revolver will leave them bleeding with 60 brute damage.
One shot from a mini rad bow will leave them irradiated, confused, and poisoned.

Two shots from a revolver will leave your target in crit.
Two shots from a mini rad bow will leave them even more irradiated, even more confused, and an RNG chance to make them collapse from radiation exposure.

So it's obvious that from two pumps of this weapon, you're bound to be screwed. We can pretty much place the mini rad bow and revolver in the same tier right? Well let's take another look at this.

Two shots from the mini rad bow, and you won't have a 100% chance of dying. You'll definitely be in a position which you don't want to be in, I'm not going to deny that. You'd have the same chance of escaping under those effects as bleeding out in crit.

Hell, like I mentioned above I've seen someone tank about 3 shots of a mini rad bow. It _doesn't exactly kill you._ It just leaves you in a very vulnerable state, which your attacker will advantage of. So is the muting a bad thing? No, not exactly. In fact, it's not all that bad and makes it a 11-12 tc worth weapon.

But the radbow doesn't come without it's own (over buffs) faults. For one, it's a completely silent tool that doesn't leave a mess behind. The only thing you'd want to look out for is holding the ebow, the ebow recharge rate, spam clicking your target (click click click click click sound) while the ebow's recharging, or other potential witnesses. With a revolver, what you need to look out for is blood from your target and the bullet cases which drop when you fire it, making it pretty evident that there was some sort of murder at the scene of the crime.

So unlike the revolver, it doesn't leave a mess. Making it actually a bit stronger than the revolver, as long as you're dead on the target. In a way, it's like buying a bundle with a revolver and syndicate minisoap. Kill the target with almost 0 evidence against you on the crime scene. The only thing left is to dispose the body.

Now what's the point of this giant rad ebow review? What does it leave us with? Which point are we getting to?

I'm not done yet.
Now that we looked at how it does in the game, we need to take a step back and look at how long we can use these weapons.

The revolver is a 11 tc weapon that only has 6 rounds installed inside of it.
The mini radbow has an infinite amount of rounds inside of it. That's the problem and it's a big one, because like Fluffy said earlier, when they got a massive murderboner and went on a giant killstreak, that sort of situation needs to have some sort of counter for.

I can talk about that topic more later but, with how the ebow works currently that sides needs to be balanced.

Now there's a few ways we could do it and I have some of my own suggestions.

1. We make an overheating system. Each shot adds towards this variable called "overheat rate", and within `/Process()` it checks for whether there's an existing overheat rate. What it does, is signal the "anti-overheat" variable. Within process() there'll be a separate check ahead for that, and if it's true then it will reduce the overheat rate by one but as well raise the overheation gauge. If the overheation gauge is raised to.. let's say 13-20, then it will run an anti-overheat breach proc. This proc will make the gun overall un usable for a short span of time. For example, maybe running `spawn(300*timesoverheated)`. Timesoverheated will have to be a separate variable, which will increase when anti overheat breach is called. Therefore, the more you use the gun, the higher chance of a longer cooldown making it null. Further suggestion, but only optional to add in, making it spew out radiation (like an advanced energy gun) but, if you buy something (with the same amount of tc as a silencer) then it won't let out radiation at all. This all means it'd run it's own longer recharge rates, restricting people from running around and going ham. If it seems complicated basically, the more times you use it, the more times it overheats, and the more times you don't want to use it.
2. We make it rechargeable like laser/energy guns. Meaning traitors have to break into security points (or use free to access ones in science's test range) to recharge their gun. Maybe after around 15 shots? We have to think rationally here and not in the mind of 'but it's suppose to be op and longlasting'. It's not like I'm saying it should be SIX worth shots, so it's paired up to a revolver.

and yeah, thats it. thanks for reading dunt f3rget to leaf a like & subsrcirbe

Weapons like these need some restraint for game balance. Even if there wasn't a PR to make it silence on the second shot, I'd still suggest making it less than an infinite __slightly lesser__ revolver. Let's not fool ourselves here, ever since the rebase radiation damage has gotten more notably lethal.

tl;dr
click CTRL + F
put in your search box "Alright, so I took the radbow out for a test run during a traitor round"
start from there, it'll be easier.

**Comment by Super3222** [comment]
Alright, so I took the radbow out for a test run during a traitor round... from what I can say being in close range  within a closed area will make it very easy to kill a target. Sorta of like a revolver, but, it's much slower to fire. Within a fair amount of shots into my target, they eventually got knocked out or their stamina is drained. Something like that. I'm very sneaky with my shots, and like to gamble with situations here and there. But I'll let out everything from my personal opinion and experience playing the game.

Now at a fairly estimated medium range, it took me a few shots to finally get them to collapse. Maybe it was because they were wearing an engineering hardsuit, maybe not. But, if another traitor hadn't emagged telecoms then I would've been exposed pretty badly. He was screaming, A LOT.

From what I discovered, the radbow within the first 2-3 shots will almost always lock them in a position for death. Whether it be from the radiation making them exhausted (and falling down to collapse) or whether it'd be another last shot to cause the former. Honestly, I'd think it'd be a little more RNG honestly depending on the time frame of your shots (on whether they hit or miss).

I'm not going to fabricate anything here, this weapon is **pretty goddamn lethal**. But let's take a look at how Shadow is buffing it.
He's adding a muting effect, which can only happen if the target is already confused.
Confusion can happen a lot of ways, drunk people, certain chemicals, maybe brain damage.
But where it's most likely going to come from, is the first ebow shot from the gun.

So let's say we're trying to kill some random greytider (definitely not a breach of the rules) just sitting around.

So when that first shot connects, the guy is already confused and he's moving all over the place. If he's lucky, maybe he'll move in a direction to get him away from the shooter. In the time frame of the ebow recharging it self, he has enough time to scream over the comms. Meaning whether they'll have time to scream for help or not will only depend on how they would prioritize their time between the ebow-recharge time (which is decent).

Your question is probably "but what's the point of this weapon even, and why is it this way even?"
The entire point at the beginning was to get rid of the combo cafe cheese.
Making the ebow more lethal by introducing a new weapon that was outside of chemicals, which could toxicate it's target's was fascinating.
At this point, the radbow is basically just an alternative of the revolver. It's a weapon which to use, you have to gamble with it and play your cards right.

We can't sport how well this gun is already on practical RNG, that is unfair. We have to look at what it's suppose to do, and how people are going to use it. And what I see is a gun which gives you promising results in it's own way.

So let's get back to my main point here ... **the radbow will only mute targets which are already confused**. 

Now, while we're looking at the predicted gameplay strats here, let's compare it to the revolver.

One shot from a revolver will leave them bleeding with 60 brute damage.
One shot from a mini rad bow will leave them irradiated, confused, and poisoned.

Two shots from a revolver will leave your target in crit.
Two shots from a mini rad bow will leave them even more irradiated, even more confused, and an RNG chance to make them collapse from radiation exposure.

So it's obvious that from two pumps of this weapon, you're bound to be screwed. We can pretty much place the mini rad bow and revolver in the same tier right? Well let's take another look at this.

Two shots from the mini rad bow, and you won't have a 100% chance of dying. You'll definitely be in a position which you don't want to be in, I'm not going to deny that. You'd have the same chance of escaping under those effects as bleeding out in crit.

Hell, like I mentioned above I've seen someone tank about 3 shots of a mini rad bow. It _doesn't exactly kill you._ It just leaves you in a very vulnerable state, which your attacker will advantage of. So is the muting a bad thing? No, not exactly. In fact, it's not all that bad and makes it a 11-12 tc worth weapon.

But the radbow doesn't come without it's own (over buffs) faults. For one, it's a completely silent tool that doesn't leave a mess behind. The only thing you'd want to look out for is holding the ebow, the ebow recharge rate, spam clicking your target (click click click click click sound) while the ebow's recharging, or other potential witnesses. With a revolver, what you need to look out for is blood from your target and the bullet cases which drop when you fire it, making it pretty evident that there was some sort of murder at the scene of the crime.

So unlike the revolver, it doesn't leave a mess. Making it actually a bit stronger than the revolver, as long as you're dead on the target. In a way, it's like buying a bundle with a revolver and syndicate minisoap. Kill the target with almost 0 evidence against you on the crime scene. The only thing left is to dispose the body.

Now what's the point of this giant rad ebow review? What does it leave us with? Which point are we getting to?

I'm not done yet.
Now that we looked at how it does in the game, we need to take a step back and look at how long we can use these weapons.

The revolver is a 11 tc weapon that only has 6 rounds installed inside of it.
The mini radbow has an infinite amount of rounds inside of it. That's the problem and it's a big one, because like Fluffy said earlier, when they got a massive murderboner and went on a giant killstreak, that sort of situation needs to have some sort of counter for.

I can talk about that topic more later but, with how the ebow works currently that sides needs to be balanced.

Now there's a few ways we could do it and I have some of my own suggestions.
1. We make an overheating system. Each shot adds towards this variable called "overheat rate", and within `/Process()` it checks for whether there's an existing overheat rate. What it does, is signal the "anti-overheat" variable. Within process() there'll be a separate check ahead for that, and if it's true then it will reduce the overheat rate by one but as well raise the overheation gauge. If the overheation gauge is raised to.. let's say 13-20, then it will run an anti-overheat breach proc. This proc will make the gun overall un usable for a short span of time. For example, maybe running `spawn(300*timesoverheated)`. Timesoverheated will have to be a separate variable, which will increase when anti overheat breach is called. Therefore, the more you use the gun, the higher chance of a longer cooldown making it null. Further suggestion, but only optional to add in, making it spew out radiation (like an advanced energy gun) but, if you buy something (with the same amount of tc as a silencer) then it won't let out radiation at all. This all means it'd run it's own longer recharge rates, restricting people from running around and going ham. If it seems complicated basically, the more times you use it, the more times it overheats, and the more times you don't want to use it.
2. We make it rechargeable like laser/energy guns. Meaning traitors have to break into security points (or use free to access ones in science's test range) to recharge their gun. Maybe after around 10-15 shots? We have to think rationally here and not in the mind of 'but it's suppose to be op and longlasting'. It's not like I'm saying it should be SIX worth shots, so it's paired up to a revolver.

Weapons like these need some restraint for game balance. Even if there wasn't a PR to make it silence on the second shot, I'd still suggest making it less than an infinite **slightly lesser** revolver. Let's not fool ourselves here, ever since the rebase radiation damage has gotten notably more lethal.

**Comment by ParagonPrime** [comment]
A dampener on continued use _**within a timeframe**_ would be very useful. Using rechargers is bleh- Syndicate equipment should not have to rely on NanoTrasen equipment, and moreover, the stability of the power grid. Keep in mind the radbow is designed for _covert_ operatives, not people who should be breaking windows to security checkpoints or generally being seen where they're not supposed to be.

I enjoy the radbow's use as a sort of quiet assassination weapon which **notably does not automatically signal murder.** Even if the body ends up being discovered somehow, it can seem like a workplace accident- especially in genetics or engineering.

Make an overheat function. Maybe an upgrade that lowers the overheat. But keep the infinite self-recharge, and at a decent rate.

**Comment by ShadowDeath6** [comment]
This isn't better than a revolver by any means. It's circumstancially good and nerfing it to where it fires even slower and god forbid **backfires against the user** will just make it less viable than it already is. The main question that goes through people's minds when picking 2 items of the same value is "how easily can I dispatch my target and get out scot free?" I understand that not leaving a bloody mess on the ground is extremely useful, but its killing effectiveness is less than that of other weapons. This is intended to be a stealthy, quick kill on unsuspecting targets or in a one on one situation.

**Comment by Daman453** [comment]
I used it, i got one shot off, he was able to run away, fireaxe to the head isn't tasty. Ebow sounds good now! Better then what it was, as seen as this merges, i will test it out.

**Comment by ghost** [comment]
An excellent summary, super, but you are forgetting a key point-

Nobody likes the revolver. We were considering how to nerf before the rebase.

In some ways this weapon is WORSE than the revolver, because even if your target survives he'll have horrible mutations, as compared to slap 2 patches and call it a day.

The revolver is easy to get extra ammo for.

The ebow's infinite ammo is fine, and it's recharge rate is slow enough to be fair.

Now, the way the goon radbow is used (which is what this is based off of) is that it's a good weapon for hitting your target unexpectedly and from dark areas.

I would say, rather than having an auto-mute which no other stunning weapon does, make the message onhit smaller, or nonexistent, so stealthy attacks are plausible.

**Comment by FluffySurvivor** [comment]
hold on

The ebow is in the same tier as the revolver, it's the stealthy alternative. 

This buff is meant to accentuate the stealth identity and to actually put it in revolver-tier. The mute is not instant, and it has a lot of counterplay. If you get shot once, you know you should run/dodge the next hit/disarm the attacker. 

Ebow is fine, this pr is fine

**Comment by ParagonPrime** [comment]
i'd be good with either the no message and totally silent bow or the mute, honestly- both are useful in their own situations

**Comment by oisin100** [comment]
Before i merge this i would like to see the cooldown idea implemented.

**Comment by ShadowDeath6** [comment]
I'm not going to add a cooldown, that makes it marginally worse than the revolver(or any other gun because guns fire faster and have no cooldown) and probably worth around 8 tc.

**Comment by ShadowDeath6** [comment]
I think I've made my point clear on how I feel about these changes, I doubt it'll influence the traitor meta too heavily to implement this. If it really becomes a problem I'll either revert this change or add a cooldown, but imposing such a nerf on the bow before it's actually tested live is probably a bad idea.
